R/pkg/DESCRIPTION

```diff
@@ -54,5 +54,5 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
This happens automatically. Ideally the build doesn't modify source files -- do you know any way to make it not do this?